pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -34,7 +34,7 @@ const minimumDockerAPIVersionWithPullByID = "1.22"
 // All errors here are fatal.
 func (c *NodeConfig) EnsureKubeletAccess() {
 	if _, err := os.Stat("/var/lib/docker"); os.IsPermission(err) {
-		c.HandleDockerError("Unable to view the /var/lib/docker directory - are you running as root?")
```

Comment:
seems like we should be moving away from these checks as we lean more on cri, not making them unconditionally fatal

Reply:
> seems like we should be moving away from these checks as we lean more on cri, not making them unconditionally fatal

This is a pre-existing move.  Not something net new.

Reply:
or at least move this into EnsureDocker or protect with `if c.KubeletServer.ContainerRuntime == "docker" { ... }`